### PR TITLE
Build tooling

### PR DESCRIPTION
**Pull request by danni**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,6 +19,8 @@
     "style-loader": "^0.12.3"
   },
   "devDependencies": {
-    "webpack-dev-server": "^1.10.1"
+    "flow-bin": "~0.13.1"
```
Comment (DanielHeath) [inline package.json]: Missing a trailing comma here

File: script/typecheck [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 set -euo pipefail
 IFS=$'\n\t'
+PATH=$PATH:`npm bin`
 
 if ! which flow ; then
     echo "You need 'flow' installed (available via brew on osx)"
     exit 1
 fi
 
+jsxhint
```
Comment (DanielHeath) [inline script/typecheck:11]: What installs this binary?

Comment (danni) [inline script/typecheck:11]: The jxhint node package.

Comment (DanielHeath) [inline script/typecheck:11]: worth adding to the readme?

**Comment by DanielHeath** [comment]
:ship:

**Pull request merged**